packages/shared/providers.yaml

```diff
@@ -3700,6 +3700,31 @@ microsoft-power-bi:
         base_url: https://api.powerbi.com
     docs: https://docs.nango.dev/integrations/all/microsoft-power-bi
 
+mindbody:
+    display_name: Mindbody
+    categories:
+        - fitness
+    auth_mode: API_KEY
+    proxy:
+        base_url: https://api.mindbodyonline.com/public/v6
+        headers:
+            API-Key: ${apiKey}
+            siteId: ${connectionConfig.siteId}
```

Comment:
You might have left this;  `-H 'authorization: {staffUserToken}'`

Reply:
added

Reply:
This header is a connectionConfig, might be good to have it as `${connectionConfig.staffUserToken}`.

Reply:
@hassan254-prog I've updated it to use the connection config 👌🏾